### DISCUSSION TITLE
My application is always rejected by pre-check.

### DISCUSSION THREAD

**Discussion by scd-scd** [community/community#42588] [GitHub Education]
I have completed my GitHub user profile with my relevant information.However,it always says as the following:

1.Your application cannot be reviewed until you fix the following:
Please upload a different document to demonstrate your current academic status.,Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?,You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.

2.Your image should be one of the following:
Dated school ID

**Comment by hiten36** [comment]
Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field.

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159

**Reply by scd-scd** [reply]
Thanks for answer.
However, neither website nor Twitter does I have.
All except these I have set up, but it still says that I am significantly more likely to be verified if I have completed my GitHub user profile with my full name and a short bio.

**Reply by scd-scd** [reply]
Should I use my original name which is written in other language as my GitHub name?
